packages/edge-login-ui-rn/src/native/components/exportAPIComponents/ChangePasswordScreen.js

```diff
@@ -35,19 +36,26 @@ class ChangePasswordScreen extends Component<Props> {
 
   componentWillMount () {
     setLocal(this.props.locale, this.props.language)
+
+    // Hack to make Flow happy, since it expects these objects to exist:
+    const missing: any = void 0
+
+    const imports: Imports = {
+      accountObject: this.props.account,
+      accountOptions: missing,
+      callback: missing,
```

Comment:
If these can be missing, they should probably be marked optional in the `Imports` type. Doing this created a lot of Flow errors, though, so I just used `any` to make Flow happy for now.